modules/navigation_mesh_generator/config.py

```diff
@@ -0,0 +1,16 @@
+def can_build(env, platform):
+    return True
+
+
+def configure(env):
+    pass
+
+
+def get_doc_classes():
+    return [
+        "NavigationMeshGenerator",
```

Comment:
This `NavigationMeshGenerator` is based on `Recast`, should `Recast` be in the name of the module?

Reply:
My understanding is that it's not a 1:1 port of Recast, so no.

Reply:
NavigationMeshGenerator does more like the entire SceneTree geometry parsing for the source geometry. ReCast is only the part that turns this source geometry into a navmesh for 3D.